.circleci/config.yml

```
@@ -142,31 +142,42 @@ jobs:
           command: |
             if [[ $CIRCLE_TAG ]]; then
                 # This is a tagged release, version has been handled upstream
-                export RELEASE_VERSION=$CIRCLE_TAG
-            # Otherwise, relies on a dev version like "1.2.1.dev" by default
-            elif [[ "$CIRCLE_BRANCH" == feature/* ]]; then
-                # This is a feature branch
-                export RELEASE_VERSION=$CIRCLE_BUILD_NUM+${CIRCLE_BRANCH#*/}
+                RELEASE_VERSION=$CIRCLE_TAG
             else
-                # This is a simple development build
-                export RELEASE_VERSION=$CIRCLE_BUILD_NUM
+                # Otherwise, relies on a dev version like "1.2.1.dev" by default
+                # Read version from __init__.py
+                BASE_VERSION=$(python -c "from udata import __version__; print(__version__)")
+                if [[ "$CIRCLE_BRANCH" == feature/* ]]; then
+                    # This is a feature branch
+                    BUILDNO=$CIRCLE_BUILD_NUM+${CIRCLE_BRANCH#*/}
+                else
+                    # This is a simple development build
+                    BUILDNO=$CIRCLE_BUILD_NUM
+                fi
+                # Build the release version string with the build number, like "1.2.1.dev+1234"
+                RELEASE_VERSION="${BASE_VERSION}+${BUILDNO}"
```

I think we don't have the plus sign
```suggestion
                # Build the release version string with the build number, like "1.2.1.dev1234"
                RELEASE_VERSION="${BASE_VERSION}${BUILDNO}"
```